annotation-utils/src/main/java/eu/europeana/annotation/config/AnnotationConfiguration.java

```diff
@@ -75,4 +79,11 @@
      */
     public Set<String> getAcceptedLicenceses();
 
+	/**
+	 * uses annotation.apiVersion property
+	 */
+	public String getAnnotationApiVersion();
+	
+	public String getEtagFormat();
```

Comment:
please remove this method, it must not be in configurations

Reply:
method removed